### PR TITLE
ensure remaining nibbles get pushed onto the resulting address

### PR DESCRIPTION
**Pull request by mzagrabe**
If the prefix length for an ip6.arpa zone is not a multiple of 16, then
remaining nibbles will get silently dropped.

### CODE REVIEW

**Comment by cvicente** [comment]
Hi Matt,

Thanks for this. Would you by chance have any test code to go with your changes?

**Comment by mzagrabe** [comment]
Hey Carlos,

Not yet. I'll take a look.

-m

File: lib/Netdot/Model/Zone.pm [hunk 1]
```diff
@@ -988,10 +988,18 @@ sub _dot_arpa_to_ip {
...
 	for (my $i=1; $i<=scalar(@n); $i++){
 	    $m .= $n[$i-1];
+        # If we are at a "group" boundary, then push it on.
 	    if ( $i % 4 == 0 ){
 		push @g, $m;
 		$m = "";
 	    }
+        # If we are at the last character of the address, then process
+        # what is left.
+        elsif ($i == @n) {
```
Comment (cvicente) [inline lib/Netdot/Model/Zone.pm]: Indentation isn't quite right, is it?

**Comment by mzagrabe** [comment]
I use 4 space indents with no tabs, so it could be affecting things. But I
also noticed there is no consistency between spaces and tabs, nor spacing
in general in the codebase.

Feel free to tweak it.

-m

**Comment by mzagrabe** [comment]
Thanks!

**Pull request merged**